scripts/figure_eight.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+import rospy
+from std_msgs.msg import String, Header
+from nav_msgs.msg import Odometry, Path
+from geometry_msgs.msg import Twist, PoseStamped
+import math
+import sys
+import time
+import tf
+from srslib_framework.msg import MsgUpdateToteLights
+
+
+def messageCreation(message, cmd, sC, eC, sS, eS, freq):
+    message.lightCmd = cmd
+    message.startColor.r = sC[0]
+    message.startColor.g = sC[1]
+    message.startColor.b = sC[2]
+    message.startColor.a = sC[3]
+    message.endColor.r = eC[0]
+    message.endColor.g = eC[1]
+    message.endColor.b = eC[2]
+    message.endColor.a = eC[3]
+    message.startSegment.x = sS[0]
+    message.startSegment.y = sS[1]
+    message.startSegment.z = sS[2]
+    message.endSegment.x = eS[0]
+    message.endSegment.y = eS[1]
+    message.endSegment.z = eS[2]
+    message.frequency = freq
+
+
+class FigureEight:
+	VEL_COM_TOPIC = "/sensors/odometry/velocity/cmd"
+	GOAL_TOPIC = "/path_segment"
+	LIGHT_TOPIC = "/drivers/brainstem/cmd/update_tote_lights"
+	global rospy
+
+	def __init__(self,offX, offY, rad):
```

Comment:
inconsistent formatting... use offsetX instead of offY and radius instead of rad, etc...